packages/shared/lib/integrations/scripts/webhook/webhook.manager.ts

```diff
@@ -37,11 +26,11 @@ async function execute(environmentUuid: string, providerConfigKey: string, heade
 
     const handler = handlers[`${provider.replace(/-/g, '')}Webhook`];
 
-    let res: WebhookResponse | null | void = null;
+    let res: WebhookResponse = undefined;
```

Comment:
Why undefined instead of null here?

Reply:
WebhookResponse is already equal to `{} | void`. undefined and null values are often swapped but they mean something different, in that case we expect res to be `undefined` because the `handler()` call can fail so there is no need to have `null`. It's just a type consistency, no change in logic.